superset/connectors/sqla/models.py

```diff
@@ -1232,9 +1232,9 @@ def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-ma
                     elif op == utils.FilterOperator.LESS_THAN_OR_EQUALS.value:
                         where_clause_and.append(sqla_col <= eq)
                     elif op == utils.FilterOperator.LIKE.value:
-                        where_clause_and.append(sqla_col.like(eq))
+                        where_clause_and.append(sqla_col.like(f"%{eq}%"))
```

Comment:
As `LIKE` can do much more than just a "contains" match, and because this would be a breaking change, I'd rather prefer something that allows the user to write their own matching rule and not break backwards compatibility. Do you think it would make sense to only append the leading and trailing `%` if there are no unescaped `%` characters present in the like clause?

Reply:
It seems we need more discussion about filters. Some more design may be needed here.